packages/native/darwin-arm64/LICENSE.md

```
@@ -1,4 +1,4 @@
-# MIT License
+MIT License
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)